src/mpl/configure.ac

```diff
@@ -659,6 +659,26 @@ AS_CASE([$enable_yield],
 #######################################################################
 
 
+#######################################################################
+## START OF MISC CODE
+#######################################################################
+
+AC_CHECK_HEADERS(sys/sysinfo.h unistd.h)
+
+AC_CHECK_FUNCS(get_nprocs)
+AC_CHECK_DECLS([_SC_NPROCESSORS_ONLN],[],[],[[#include <unistd.h>]])
+
+if test "$ac_cv_func_get_nprocs" = "yes" -a "$ac_cv_have_decl__SC_NPROCESSORS_ONLN" = "yes"; then
+    AC_DEFINE(HAVE_MISC_GETNPROCS,1, [Define if get_nprocs or _SC_NPROCESSORS_ONLN is Vilable.])
+else
+    AC_MSG_WARN([Undefined get_nprocs nor _SC_NPROCESSORS_ONLN])
```

Comment:
I would remove the warnings as well. Most likely the warnings will be buried in the configure output no one would notice. And when they do, it is out of context, no one would understand what for. If you must add warning, add a runtime error in the code.

Reply:
Removed the warning.